crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1606,8 +1606,7 @@ impl<'db> Type<'db> {
                     .instance_member(db, name)
             }
             Type::Callable(CallableType::General(_)) => {
-                // TODO: Implement static member lookup for general callable types
-                Symbol::todo("static member lookup on general callable type").into()
+                KnownClass::Object.to_instance(db).member(db, name)
```

Comment:
I think this needs to be
```suggestion
                KnownClass::Object.to_instance(db).instance_member(db, name)
```

Probably doesn't matter since you special case general callable types in `member_lookup_with_policy` below, but it would matter if this function gains more call sites.

Reply:
Oh right, thanks for catching that.